### DISCUSSION TITLE
Apex account / saved data sync

### DISCUSSION THREAD

**Discussion by inggah33** [janstehno/cotw-companion#61] [Ideas]
Hi, in my opinion the app is very good but it can always be improved. What I would do would be to link the apex account with the app to have all the information of my account, maps, data, trophies, activity, total number of animals in your active map in console / PC without specifications of the same only to know if they reappear when you hunt them I say this because I am a console player xbox series x I have the full game and when I hunt bigger male in a habitat of 2 males and 3 females I never get 2 males in that habitat so I would like to know the number of animals by class or in general to know if they are elsewhere, are bug or directly do not reappear. That's all for the moment, my pleasure and thank you

> Translated with DeepL.com

hola en mi opinion la app esta muy bien aunque siempre se puede mejorar. Lo que yo haria seria vincular la cuenta de apex con la app para tener toda la informacion de mi cuenta, mapas, datos, trofeos, actividad, numero total de animales en tu mapa activo en consola/pc sin especificaciones de los mismos solo para saber si reaparecen cuando los cazas digo esto por que soy jugador de consola xbox series x tengo el juego completo y cuando  cazo macho mas grande en un habitad de 2 machos y 3 hembras nunca me vuelven a salir 2 machos en ese habitad por eso me gustaria saber el numero de animales por clase o en general paras saber si estan en otro lado, estan bug o directamente no reaparecen.
Eso es todo de momento un placer y gracias

> Original

**Comment by janstehno** [comment]
Unfortunately, this is not possible right now. It was suggested a lot of times. However, the developers of the game do not provide the middleware I could use to implement this connection and synchronization between the app and the game. So for now this won't be implemented. Not that I do not want to implement it, I just can not.
